examples/authentication.py

```diff
@@ -82,3 +82,49 @@ def token_gen_call(username, password):
     if mockpassword == password and mockusername == username: # This is an example. Don't do that.
         return {"token" : jwt.encode({'user': username, 'data': 'mydata'}, secret_key, algorithm='HS256')}
     return 'Invalid username and/or password for user: {0}'.format(username)
+
+# JWT AUTH EXAMPLE #
+replace_this = False
+config = {
+    'jwt_secret': 'super-secret-key-please-change',
+    # token will expire in 3600 seconds if it is not refreshed and the user will be required to log in again
```

Comment:
PEP seems to indicate that "If a comment is short, the period at the end can be omitted."

I usually start comments with lower case, but yeah, that doesn't seem compliant. I'll change this then.